clang/test/dpct/cmake_migation/case_013/expected.txt

```diff
@@ -0,0 +1,19 @@
+project(foo-bar LANGUAGES CXX )
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsycl")
+find_program(dpct_bin_path NAMES dpct PATHS)
+get_filename_component(bin_path_of_dpct ${dpct_bin_path} DIRECTORY)
+set(dpct_cmake_file_path "${bin_path_of_dpct}/../cmake/dpct.cmake")
+include(${dpct_cmake_file_path})
+
+set(cufile file1.dp.cpp)
+dpct_compile_sycl_code(cubinfile_1 ${cufile})
+dpct_compile_sycl_code( cubinfile_2 
+${cufile}
+)
+
+dpct_compile_sycl_code(cubinfiles_1 file1.dp.cpp file2.dp.cpp)
+dpct_compile_sycl_code(cubinfiles_2 file1.dp.cpp 
+file2.dp.cpp)
+
+dpct_compile_sycl_code(cubinfile_1 file1.dp.cpp OPTIONS
+  -O3 -Xptxas -v --use_fast_math)
```

Comment:
"PTIONS  -O3 -Xptxas -v --use_fast_math" is specific to cuda, not needed to be appeared in SYCL version cmake scripts.